src/debugging/netSdk/NetSdkDebugHelper.ts

```diff
@@ -0,0 +1,51 @@
+import { commands } from "vscode";
+import { NetChooseBuildTypeContext, netContainerBuild } from "../../scaffolding/wizard/net/NetContainerBuild";
+import { unresolveWorkspaceFolder } from "../../utils/resolveVariables";
+import { DockerDebugScaffoldContext } from "../DebugHelper";
+import { DockerDebugConfiguration } from "../DockerDebugConfigurationProvider";
+import { NetCoreDebugHelper, NetCoreDebugScaffoldingOptions } from "../netcore/NetCoreDebugHelper";
+
+const netContainerSdkBuildSymbol = 'dotnet-container-sdk: debug';
+export class NetSdkDebugHelper extends NetCoreDebugHelper {
+
+    public async provideDebugConfigurations(context: DockerDebugScaffoldContext, options?: NetCoreDebugScaffoldingOptions): Promise<DockerDebugConfiguration[]> {
+
+        const providers: DockerDebugConfiguration[] = [];
+
+        const netCoreBuildContext: NetChooseBuildTypeContext = {
+            ...context.actionContext,
+            scaffoldType: 'debugging',
+            workspaceFolder: context.folder,
+        };
+
+        await netContainerBuild(netCoreBuildContext);
+
+        if (netCoreBuildContext?.containerBuildOptions === 'Use .NET SDK') {
```

Comment:
Use the constant declaration in NetSdkChooseBuildStep.ts rather than cloning a new string